src/components/templates/WidgetsTemplate.tsx

```diff
@@ -2,43 +2,46 @@ import React, {FunctionComponent, useEffect} from 'react'
 
 import {useStore} from '../../store/storeConfig'
 import RootStore from '../../store/root.store'
-import {observer} from 'mobx-react-lite'
+import {observer} from 'mobx-react-lite';
 import {Grid} from '../atoms'
 import AnyWayCard from '../molecules/AnyWayCard'
 import CountByYearBarWidget from '../molecules/CountByYearBarWidget'
 import CountByTypePieWidget from '../molecules/CountByTypePieWidget'
 import CountBySeverityPieWidget from '../molecules/CountBySeverityPieWidget'
-// import HeatMap from '../molecules/HeatMap'
+import HeatMap from '../molecules/HeatMap'
 import LocationMap from '../molecules/LocationMap'
 import ErrorBoundary from '../atoms/ErrorBoundary';
+import { uniquePoints } from "../../utils/utils";
 
 interface IProps {
   id: number | null;
 }
 
 const getWidgetByType = (widget: any) => {
   let widgetComponent;
-
   switch (widget.name) {
     case 'most_severe_accidents': {
       widgetComponent = <LocationMap data={widget.data} center={{lat: 32.0853, lng: 34.7818}}/>;
       break;
     }
     case 'accidents_heat_map': {
```

Comment:
is it need to be  `accidents_heat_map'` or `'most_severe_accidents_heatmap'`?

Reply:
accidents_heat_map - it's what coming from the server
most_severe_accidents_heatmap - this one only on mock

Reply:
we should change mock
`most_severe_accidents_heatmap` => `most_severe_accidents_heatmap`